plugins/org.jboss.tools.jst.web.ui/src/org/jboss/tools/jst/web/ui/internal/editor/jspeditor/xpl/StyledTextDropTargetEffect.java

```diff
@@ -155,6 +155,7 @@ public void dragOver(DropTargetEvent event) {
 		
 		Point mousePoint = text.getDisplay().map(null, text, event.x, event.y);
 		Point newOffsetPoint = text.getLocationAtOffset(newOffset);
+		if (newOffset < 0) return;
```

Comment:
We should check if newOffset<0 before (!) Point newOffsetPoint = text.getLocationAtOffset(newOffset);
Not after.